### PR TITLE
REST resource to get the list of NS

### PR DESCRIPTION
**Pull request by cdiop**

### CODE REVIEW

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java [hunk 1]
```diff
@@ -1377,10 +1377,10 @@ public BooleanWrapper setNodeSourcePingFrequency(int frequency, String sourceNam
     }
 
     /**
-     * {@inheritDoc}
+     * Gives list of existing Node Sources
+     * @return list of existing Node Sources
      */
-    @Deprecated
-    public List<RMNodeSourceEvent> getNodeSourcesList() {
+    public List<RMNodeSourceEvent> getExistingNodeSourcesList() {
```
Comment (sonarqube-activeeon) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java]: MAJOR Add the "@Override" annotation above this method signature [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [RMProxyUserInterface.java](https://github.com/ow2-proactive/scheduling/blob/7b6b398485e4164f8383c77749fa907250607f37/rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMProxyUserInterface.java): Replace all tab characters in this file by sequences of white-spaces. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00105)

**Comment by yinan-liu** [comment]
Code reviewed !

**Comment by mbenguig** [comment]
Ok for me

**Pull request merged**